packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/deploy/cdk-deploy-telemetry.integtest.ts

```diff
@@ -0,0 +1,198 @@
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import { integTest, withDefaultFixture } from '../../../lib';
+
+jest.setTimeout(2 * 60 * 60_000); // Includes the time to acquire locks, worst-case single-threaded runtime
+
+integTest(
+  'cdk deploy with telemetry data',
+  withDefaultFixture(async (fixture) => {
+    const telemetryFile = path.join(fixture.integTestDir, 'telemetry.json');
+
+    // Deploy stack while collecting telemetry
+    await fixture.cdkDeploy('test-1', {
+      telemetryFile,
+    });
+    const json = fs.readJSONSync(telemetryFile);
+    expect(json).toEqual([
```

Comment:
There's a log of `except.anything` here...can we make it more strict? or on the contrary, if all you really want to test is the existence of 3 types of events, just assert for that then? 

I think its fine as long as you unit test the concrete values somewhere else.